sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextDocumentInput.java

```diff
@@ -36,7 +36,6 @@
     public TextDocumentInput(String id, String text) {
         this.id = id;
         this.text = text;
-        this.language = "en";
```

Comment:
Sorry I was wrong when I suggested this, I didn't realize that this TextDocumentInput constructor didn't take language. I'm confused though: it looks like in [master](https://github.com/Azure/azure-sdk-for-java/blob/fcdaff1ebc84a6f660f94c54038c2c34fef76e0c/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextDocumentInput.java#L50) you had two constructors: one that took language, one that did not. 

I think you should still have two constructors here like in master. For the one that doesn't take language as a parameter, it should set this.language = "en", since that is the default. Otherwise, this.language = language

Reply:
if in the future, the default lanauge is change to 'es`. we need to update it to `es`. Since service team take default value in service side, we should not add what they are maintaining here. So keep no-assigned in SDK should be an better idea.